playbooks/templates/rax-maas/neutron_ovs_agent_check.yaml.j2

```diff
@@ -1,14 +1,15 @@
 {% from "templates/common/macros.jinja" import get_metadata with context %}
 {% set label = "neutron_ovs_agent_check" %}
 {% set check_name = label+'--'+inventory_hostname %}
+{% set service_host = ansible_nodename + ansible_local['maas']['general']['local_suffix'] | default('') %}
 type        : agent.plugin
 label       : "{{ check_name }}"
 period      : "{{ maas_check_period_override[label] | default(maas_check_period) }}"
 timeout     : "{{ maas_check_timeout_override[label] | default(maas_check_timeout) }}"
 disabled    : "{{ (check_name is match(maas_excluded_checks_regex)) | ternary('true', 'false') }}"
 details     :
     file    : run_plugin_in_venv.sh
-    args    : ["{{ maas_plugin_dir }}/neutron_ovs_agent_check.py"]
+    args    : ["{{ maas_plugin_dir }}/neutron_ovs_agent_check.py", "--protocol", "{{ neutron_client_protocol }}", "--host", "{{ service_host }}", "--fqdn", "{{ ansible_fqdn }}", "{{ internal_vip_address }}"]
```

Comment:
Is there a reason for both `host` and  `fqdn` here? These values are identical, at least on the OSP side. However, I suspect the same on OSA:

`"--host", "overcloud-controller-1.localdomain", "--fqdn", "overcloud-controller-1.localdomain"`

Reply:
This is to follow the same pattern as other neutron agent's checks:

https://github.com/rcbops/rpc-maas/blob/master/playbooks/templates/rax-maas/neutron_linuxbridge_agent_check.yaml.j2#L12